docs/streaming.md

```diff
@@ -14,43 +14,40 @@ client while the page is being generated.
     streaming will be the easiest way to get going. Streaming can give you
```

Comment:
Improve these docs:
- Use the same example/output for for both sync and async examples
- Remove/shorten the section on callables/lambda and generators.